### PR TITLE
Add policy id param

### PR DESCRIPTION
**Pull request by mccraveiro**
closes https://github.com/pagarme/credenciamento/issues/246

### CODE REVIEW

File: __tests__/adapters/pagarme/risk-analisys.unit.js [hunk 1]
```diff
@@ -185,3 +185,14 @@ test('the adapter must return policy id 5 for corporations', () => {
 
   expect(riskAnalysis).toHaveProperty('member')
 })
+
+test('the adapter must return the chosen policy id', () => {
+  const customPolicyId = 42
```
Comment (flyingluscas) [inline __tests__/adapters/pagarme/risk-analisys.unit.js]: :eyes:

File: src/adapters/pagarme/risk-analysis.js [hunk 2]
```diff
@@ -3,18 +3,24 @@ const {
...
 
-const isIndividual = propEq('document_type', 'cpf')
+const isIndividual = pathEq(['recipient', 'document_type'], 'cpf')
 
-const policyId = ifElse(isIndividual, always(6), always(5))
+const getPolicyId = cond([
+  [prop('policyId'), prop('policyId')],
+  [isIndividual, always(6)],
+  [T, always(5)],
+])
```
Comment (macabeus) [inline src/adapters/pagarme/risk-analysis.js]: Do we really need to use Ramda for this?

Comment (macabeus) [reply src/adapters/pagarme/risk-analysis.js]: Why not

```js
const getPolicyId = riskAnalysis => {
  if (riskAnalysis. policyId) {
    return riskAnalysis. policyId
  }

  if (isIndividual(riskAnalysis)) {
    return 6
  }

  return 5
}
```

Comment (uselessdev) [reply src/adapters/pagarme/risk-analysis.js]: > Do we really need to use Ramda for this?

Yes ✨

File: src/adapters/pagarme/risk-analysis.js [hunk 3]
```diff
@@ -3,18 +3,24 @@ const {
...
+  prop,
+  T,
 } = require('ramda')
 
-const isIndividual = propEq('document_type', 'cpf')
+const isIndividual = pathEq(['recipient', 'document_type'], 'cpf')
 
-const policyId = ifElse(isIndividual, always(6), always(5))
+const getPolicyId = cond([
+  [prop('policyId'), prop('policyId')],
```
Comment (uselessdev) [inline src/adapters/pagarme/risk-analysis.js]: Why two `prop('policyId')`?

File: src/adapters/pagarme/risk-analysis.js (hunk 2 above)
Comment (flyingluscas) [reply src/adapters/pagarme/risk-analysis.js]: Both ways are great, but the file already uses Ramda to build the spec, shouldn't we follow the pattern?

File: src/adapters/pagarme/risk-analysis.js (hunk 3 above)
Comment (flyingluscas) [reply src/adapters/pagarme/risk-analysis.js]: @uselessdev You're right! Shouldn't this be `[has('policyId'), prop('policyId')]`??

File: src/adapters/pagarme/risk-analysis.js (hunk 2 above)
Comment (macabeus) [reply src/adapters/pagarme/risk-analysis.js]: No

Comment (victorteodoro) [reply src/adapters/pagarme/risk-analysis.js]: Ramda's the language now. Ramda is beautiful language :heart: I'd get worried if people started using javascript.

**Pull request merged**